cdb2api/cdb2api.c

```diff
@@ -1965,9 +1967,15 @@ static int cdb2_send_query(cdb2_hndl_tp *hndl, SBUF2 *sb, char *dbname,
 
     if (env_tz == NULL) {
         env_tz = DB_TZNAME_DEFAULT;
-    }
+    } else if (strcmp(env_tz, "localtime") == 0) {
```

Comment:
Make those calls here, once?

https://github.com/bloomberg/comdb2/blob/master/cdb2api/cdb2api.c#L95

I'm not a huge fan of calls that set state outside of comdb2's domain, but I guess this is the portable way to do it...